ollama/_client.py

```diff
@@ -245,6 +245,29 @@ def chat(
       stream=stream,
     )
 
+  def embed(
+    self,
+    model: str = '',
+    input: Union[str, Sequence[AnyStr]] = '',
+    truncate: bool = True,
+    options: Optional[Options] = None,
+    keep_alive: Optional[Union[float, str]] = None,
+  ) -> Mapping[str, Any]:
+    if not model:
+      raise RequestError('must provide a model')
+
+    return self._request(
+      'POST',
+      '/api/embed',
+      json={
+        'model': model,
+        'input': input,
+        'truncate': truncate,
+        'options': options or {},
+        'keep_alive': keep_alive,
+      },
+    ).json()
+
   def embeddings(
```

Comment:
this can be marked deprecated with https://docs.python.org/3/library/exceptions.html#DeprecationWarning

Reply:
will make note of this in docs pr